pymtl/tools/translation/verilator_wrapper.templ.py

```diff
@@ -91,7 +94,7 @@ def elaborate_logic( s ):
     verilator_vcd_file = ""
     if s.vcd_file:
       filen, ext         = os.path.splitext( s.vcd_file )
-      verilator_vcd_file = '{{}}.verilator1{{}}'.format(filen, ext)
+      verilator_vcd_file = '{{}}.verilator{{}}'.format(filen, ext)
```

Comment:
@cbatten Why did you add a 1 here https://github.com/cornell-brg/pymtl/commit/15926e9c87f1fc76a1335e63930a138db3ca71aa?